tutorials/2 From components to chip/C. My first full quantum chip design/2.21 Design a 4 qubit full chip.ipynb

```diff
@@ -30,7 +30,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
This note book has been executed.  You can tell by the numbers in the  square brackets.  In Jupyter notebook, we usually shutdown the kernel and clear the output before committing the file in tutorial directory.